torch/nn/modules/normalization.py

```diff
@@ -141,20 +141,21 @@ class LayerNorm(Module):
         >>> output = m(input)
     """
     __constants__ = ['normalized_shape', 'eps', 'elementwise_affine']
-    normalized_shape: _shape_t
+    normalized_shape: Tuple[int, ...] 
     eps: float
     elementwise_affine: bool
 
     def __init__(self, normalized_shape: _shape_t, eps: float = 1e-5, elementwise_affine: bool = True) -> None:
         super(LayerNorm, self).__init__()
         if isinstance(normalized_shape, numbers.Integral):
-            normalized_shape = (normalized_shape,)
-        self.normalized_shape = tuple(normalized_shape)
+            # mypy error: incompatible types in assignment
+            normalized_shape = (normalized_shape,)  # type: ignore[assignment]
+        self.normalized_shape = tuple(normalized_shape)  # type: ignore[arg-type]
```

Comment:
normalized_
```suggestion
            normalized_shape_ = (normalized_shape,)
        self.normalized_shape = tuple(normalized_shape_)
```

Reply:
also wondering do we even need the second `tuple()` wrapper?

Reply:
I guess the second `tuple()` is to make sure `self.normalized_shape` is a tuple and not a list, for instance.

Reply:
Also, I do not think the suggested change works as expected. If the input arg `normalized_shape` is a list, then the if block will never be executed and `normalized_shape_` will not be defined.